client/capstone-app/package.json

```diff
@@ -19,7 +19,7 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-leaflet": "^4.2.1",
-    "react-router-dom": "^6.24.0",
+    "react-router-dom": "^6.24.1",
```

Comment:
Why are these versions changing?

Reply:
This was the latest version, and I wanted to keep the project updated